### ISSUE TITLE
Import error for M2Crypto when starting catalog on apache

### ISSUE THREAD

**Issue by adborden** [GSA/data.gov#389]
Working on sandbox, seeing this error when starting apache.

```
[Fri Aug 17 18:05:23.023938 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811] mod_wsgi (pid=10052): Exception occurred processing WSGI script '/etc/ckan/apache.wsgi'.
[Fri Aug 17 18:05:23.023964 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811] Traceback (most recent call last):
[Fri Aug 17 18:05:23.023981 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/etc/ckan/apache.wsgi", line 10, in <module>
[Fri Aug 17 18:05:23.024014 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     application = loadapp('config:%s' % config_filepath)
[Fri Aug 17 18:05:23.024032 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
[Fri Aug 17 18:05:23.024055 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     return loadobj(APP, uri, name=name, **kw)
[Fri Aug 17 18:05:23.024072 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 272, in loadobj
[Fri Aug 17 18:05:23.024091 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     return context.create()
[Fri Aug 17 18:05:23.024111 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
[Fri Aug 17 18:05:23.024134 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     return self.object_type.invoke(self)
[Fri Aug 17 18:05:23.024148 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 146, in invoke
[Fri Aug 17 18:05:23.024165 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     return fix_call(context.object, context.global_conf, **context.local_conf)
[Fri Aug 17 18:05:23.024179 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan/lib/python2.7/site-packages/paste/deploy/util.py", line 56, in fix_call
[Fri Aug 17 18:05:23.024192 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     val = callable(*args, **kw)
[Fri Aug 17 18:05:23.024209 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan-new/src/ckan/ckan/config/middleware.py", line 118, in make_app
[Fri Aug 17 18:05:23.024222 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     who_parser.parse(open(app_conf['who.config_file']))
[Fri Aug 17 18:05:23.024239 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan-new/lib/python2.7/site-packages/repoze.who-1.0.18-py2.7.egg/repoze/who/config.py", line 80, in parse
[Fri Aug 17 18:05:23.024257 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     obj = self._makePlugin(name, IPlugin, options)
[Fri Aug 17 18:05:23.024270 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan-new/lib/python2.7/site-packages/repoze.who-1.0.18-py2.7.egg/repoze/who/config.py", line 37, in _makePlugin
[Fri Aug 17 18:05:23.024282 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     obj = _resolve(name)
[Fri Aug 17 18:05:23.024299 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan-new/lib/python2.7/site-packages/repoze.who-1.0.18-py2.7.egg/repoze/who/config.py", line 20, in _resolve
[Fri Aug 17 18:05:23.024325 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     return EntryPoint.parse('x=%s' % name).load(False)
[Fri Aug 17 18:05:23.024339 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan/lib/python2.7/site-packages/pkg_resources.py", line 2048, in load
[Fri Aug 17 18:05:23.024391 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     entry = __import__(self.module_name, globals(),globals(), ['__name__'])
[Fri Aug 17 18:05:23.024420 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan-new/src/pysaml2/src/s2repoze/plugins/sp.py", line 40, in <module>
[Fri Aug 17 18:05:23.024440 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     from saml2 import ecp, BINDING_HTTP_REDIRECT
[Fri Aug 17 18:05:23.024453 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan-new/src/pysaml2/src/saml2/ecp.py", line 32, in <module>
[Fri Aug 17 18:05:23.024465 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     from saml2.server import Server
[Fri Aug 17 18:05:23.024497 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan-new/src/pysaml2/src/saml2/server.py", line 29, in <module>
[Fri Aug 17 18:05:23.024520 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     from saml2.entity import Entity
[Fri Aug 17 18:05:23.024543 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan-new/src/pysaml2/src/saml2/entity.py", line 4, in <module>
[Fri Aug 17 18:05:23.024557 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     from saml2.metadata import ENDPOINTS
[Fri Aug 17 18:05:23.024573 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan-new/src/pysaml2/src/saml2/metadata.py", line 14, in <module>
[Fri Aug 17 18:05:23.024588 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     from saml2.sigver import pre_signature_part
[Fri Aug 17 18:05:23.024600 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]   File "/usr/lib/ckan-new/src/pysaml2/src/saml2/sigver.py", line 28, in <module>
[Fri Aug 17 18:05:23.024611 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811]     import M2Crypto
[Fri Aug 17 18:05:23.024635 2018] [:error] [pid 10052:tid 140661206644480] [remote 10.0.101.57:52811] ImportError: No module named M2Crypto
```

**Comment by adborden** [comment]
I would think that if pysaml2 depends on M2Crypto, it would be installed via pip?

**Comment by adborden** [comment]
`pysaml2` is installed by the software/ckan/saml2 role, so it does not exist in development.

**Comment by adborden** [comment]
This was fixed in #392
